### DISCUSSION TITLE
make the default PR target origin/master or origin/main NOT UPSTREAM!

### DISCUSSION THREAD

**Discussion by RandyMcMillan** [community/community#101745] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Not every change to a repo fork is targeted for upstream.
Make the default PR target the origin repo NOT UPSTREAM!

Also - make UPSTREAM PRs a different color!
Make the web UI LOOK DIFFERENT IN AN OBVIOUS WAY!

**Comment by spenserblack** [comment]
You might want to consider converting your repository from a fork to a "normal" repository. More info: https://github.com/orgs/community/discussions/52663
